index.html

```diff
@@ -1799,39 +1816,40 @@ <h3>Securing Verifiable Credentials</h3>
 
         <p>
 At least one securing mechanism, and the details necessary to evaluate it, MUST
-be expressed for a <a>credential</a> or <a>presentation</a> to be a
-<a>verifiable credential</a> or <a>verifiable presentation</a>; that is, to be
-<a>verifiable</a>.
+be expressed to construct a <a>verifiable credential</a> or a
+<a>verifiable presentation</a>.
         </p>
         <p>
 This specification recognizes two classes of securing mechanisms: those that use
 external proofs and those that use embedded proofs. An
 <dfn class="lint-ignore">external proof</dfn> is one that wraps an expression of
 this data model, such as via a JSON Web Token, which is elaborated on in the
-<em>Securing Verifiable Credentials using JSON Web Tokens</em> [[VC-JWT]] specification.
-An <dfn>embedded proof</dfn> is a mechanism where the proof is included in the
-data model, such as a Data Integrity Proof, which is elaborated on in <em>Verifiable
-Credential Data Integrity</em> [[VC-DATA-INTEGRITY]].
+<em>Securing Verifiable Credentials using JSON Web Tokens</em> [[VC-JWT]]
```

Comment:
For another PR, but perhaps we can make these also references to the spec dir... Anticipating additional embedded or external proofs in the future.